### PR TITLE
[UXE-3955] test: e2e edge app functions major cases

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR?
* edge app functions major test cases
* edge app function instance in rules engine

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
no
### Does it have a link on Figma?
no
<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: cypress/e2e/edge-application.cy.js
Comment (alisson-marcelo-azion) [inline cypress/e2e/edge-application.cy.js:1]: Não entendi porque a execução do testes da spec mostra apenas um testcase executado se  foram adicionados outros no mesmo describe.
Screenshot 2024-07-22 at 10 33 17

Comment (alisson-marcelo-azion) [reply cypress/e2e/edge-application.cy.js:1]: https://github.com/aziontech/azion-console-kit/actions/runs/10013469805/job/27681138563#step:4:399

Comment (alisson-marcelo-azion) [reply cypress/e2e/edge-application.cy.js:1]: O motivo é o uso do only, neste ponto: https://github.com/aziontech/azion-console-kit/pull/1516/files#diff-0a7ed627ddf31ee3d413cae98d6a67d675c99dc808a590bef0e2631d62a6faf9R424

File: cypress/e2e/functions.cy.js
Comment (matheus-sunderhus) [inline cypress/e2e/functions.cy.js:1]: renomear arquivo com nome de produto azion, mudar para edge-functions.cy.js

File: cypress/e2e/edge-application.cy.js
Comment (peterpaulo-azion) [reply cypress/e2e/edge-application.cy.js:1]: Existia um it.only ali para facilitar no desenvolvimento, acabou subindo.
Nice catch.

File: cypress/e2e/functions.cy.js
Comment (matheus-sunderhus) [reply cypress/e2e/functions.cy.js:1]: mover hooks de describe para antes dos blocos de testes ( it() ), dessa forma os hooks ficam centralizados :
describe()
  beforeeach()
  aftereach()
  it()

File: cypress/e2e/edge-application.cy.js
Comment (alisson-marcelo-azion) [reply cypress/e2e/edge-application.cy.js:1]: Sugiro dividir em dois blocos describe, um com tag run e outro sem ela.

File: cypress/e2e/functions.cy.js
Comment (peterpaulo-azion) [reply cypress/e2e/functions.cy.js:1]: done!

**Review by william-tome — APPROVED** [review]

**Review by matheus-sunderhus — APPROVED** [review]

**Pull request merged**